ibis/pandas/core.py

```diff
@@ -330,7 +381,7 @@ def execute_until_in_scope(
 
     # pass our computed arguments to this node's execute_node implementation
     data = [
-        new_scope[arg.op()] if hasattr(arg, 'op') else arg
+        get_scope_item(new_scope, arg.op()) if hasattr(arg, 'op') else arg
```

Comment:
Missing timecontext here when calling `get_scope_item`?

Reply:
Yes, this is the same bug as the previous comment, op was directly executed and not add to op result in a None in getting time context. Will fix it